src/main/java/picard/illumina/CollectIlluminaLaneMetrics.java

```diff
@@ -118,7 +119,7 @@ protected int doWork() {
             }
         }
 
-        IlluminaLaneMetricsCollector.collectLaneMetrics(RUN_DIRECTORY, OUTPUT_DIRECTORY, OUTPUT_PREFIX, laneMetricsFile, phasingMetricsFile, READ_STRUCTURE, FILE_EXTENSION == null ? "" : FILE_EXTENSION);
+        IlluminaLaneMetricsCollector.collectLaneMetrics(RUN_DIRECTORY, OUTPUT_DIRECTORY, OUTPUT_PREFIX, laneMetricsFile, phasingMetricsFile, READ_STRUCTURE, FILE_EXTENSION == null ? "" : FILE_EXTENSION, VALIDATION_STRINGENCY);
```

Comment:
Come to fgbio to develop: https://github.com/fulcrumgenomics/fgbio/blob/master/src/main/scala/com/fulcrumgenomics/cmdline/FgBioMain.scala#L54-L57

In all seriousness, does Barclay have a facility for a sub-parser (a la python's argparse)?  We've just starting using in fgbio a main parser that has tool-wide options and a sub-parser for tools-specific options and it seems promising.